Makefile

```diff
@@ -84,6 +84,15 @@ remove: install/prepare
 test: build/test-suite
 	./build/test-suite
 
+.PHONY: fuzz
```

Comment:
1. Please add these to the `.PHONY` declaration at the top.

2. Please add a note about this to the `help` target.

Reply:
Done